src/app/dim-ui/TileGrid.tsx

```diff
@@ -58,10 +60,9 @@ export function TileGridTile({
     >
       <>
         {icon}
-        <div className={styles.details}>
-          <div className={styles.title}>{title}</div>
-          {children}
-        </div>
+        <div className={clsx(styles.title, corner ? styles.withCorner : undefined)}>{title}</div>
```

Comment:
This is the recommended way to do conditional styles with `clsx` - we have a babel plugin that then rewrites it to be the most efficient inlined conditions.

```suggestion
        <div className={clsx(styles.title, { [styles.withCorner]: corner })}>{title}</div>
```